package-lock.json

```diff
@@ -10,10 +10,9 @@
             "license": "MIT",
             "dependencies": {
                 "@babel/preset-react": "^7.14.5",
-                "@fluidframework/azure-client": "^0.48.0",
-                "@fluidframework/test-client-utils": "^0.48.0",
-                "@fluidframework/test-runtime-utils": "^0.48.0",
-                "fluid-framework": "^0.48.0"
+                "@fluidframework/azure-client": "^0.48.1",
```

Comment:
Can you revert the package-lock version in this PR too?

Reply:
@tylerbutler , why?